ci/copy-unix-release-artifacts.sh

```diff
@@ -18,42 +18,34 @@ mkdir -p $OUTPUT_DIR/split-release
 
 TARBALL=daml-sdk-$RELEASE_TAG-$NAME.tar.gz
 EE_TARBALL=daml-sdk-$RELEASE_TAG-$NAME-ee.tar.gz
-bazel build //release:sdk-release-tarball-ce //release:sdk-release-tarball-ee
```

Comment:
Build commands in this file are redundant. We could leave them here and omit them in `build-release.sh`, but this script only executes on `main`, which means they then don't get built as part of the PR. Which may or may not be a good thing, I haven't made up my mind yet.